src/development.js

```diff
@@ -22,10 +22,15 @@ export default function catchErrors({ filename, components, imports }) {
         } else {
           console.error(err);
         }
-        return React.createElement(ErrorReporter, {
-          error: err,
-          filename
-        });
+        return React.createElement(ErrorReporter,
```

Comment:
Let's style it as

```js
React.createElement(ErrorReporter, Object.assign({}, reporterOptions, {
  error: err,
  filename
}))
```

This way we have less nesting. Also, `|| {}` is not necessary inside `Object.assign`—it skips undefined stuff.